src/main/webapp/WEB-INF/views/drivers/login.jsp

```diff
@@ -0,0 +1,16 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>

```

Comment:
Let's move `login.jsp` file to the upper folder `.../WEB-INF/views/login.jsp`
Check all places that linked with this page path.